test/show.jl

```diff
@@ -1432,3 +1432,8 @@ replstrcolor(x) = sprint((io, x) -> show(IOContext(io, :limit => true, :color =>
 
 # issue #30303
 @test repr(Symbol("a\$")) == "Symbol(\"a\\\$\")"
+
+# printing of bools and bool arrays
```

Comment:
As there is more than one test, maybe use `@testset` ?

Reply:
already done.